perses/rjmc/geometry.py

```diff
@@ -90,6 +89,7 @@ def propose(self, top_proposal):
         logp_proposal : float
             The log probability of the forward-only proposal
         """
+        beta = top_proposal.beta.in_units_of(units.kilocalorie_per_mole**-1)
```

Comment:
Isn't it dangerous to have a unitless `beta` hanging around alongside unit-bearing quantities?